web-app/js/portal/filter/BooleanFilterPanel.js

```diff
@@ -41,8 +41,7 @@ Portal.filter.BooleanFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
     },
 
     _formatBoxLabel: function() {
-        var label = this.filter.label.split('_').join(' ').toTitleCase();
-        return label;
+        return this.filter.getDisplayLabel();
```

Comment:
I think that `getCQL` (below) should be moved from the `FilterPanel` classes to the `Filter` class. The `FilterPanel` would need to somehow update the `Filter.value` appropriately.